src/SFA.DAS.ApprenticeCommitments.Web/Startup/SharedUiStartup.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.Apprentice.SharedUi.Zendesk;
+using System;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Startup
+{
+    public static class SharedUiStartup
+    {
+        public static IServiceCollection AddSharedUi(
+               this IServiceCollection services, ApplicationConfiguration configuration)
```

Comment:
Could the parameter be ZenDeskConfiguration instead of ApplicationConfiguration?